velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -888,13 +888,21 @@ bool waitForTaskCompletion(exec::Task* task, uint64_t maxWaitMicros) {
 }
 
 bool waitForTaskFailure(exec::Task* task, uint64_t maxWaitMicros) {
-  // Wait for task to transition to finished state.
+  // Wait for task to transition to failed state.
   if (!waitForTaskStateChange(task, TaskState::kFailed, maxWaitMicros)) {
     return false;
   }
   return waitForTaskDriversToFinish(task, maxWaitMicros);
 }
 
+bool waitForTaskAborted(exec::Task* task, uint64_t maxWaitMicros) {
+  // Wait for task to transition to aborted state.
+  if (!waitForTaskStateChange(task, TaskState::kAborted, maxWaitMicros)) {
```

Comment:
nit: consider extracting helper method to avoid code duplication

```
bool waitForTaskFailure(exec::Task* task, uint64_t maxWaitMicros) {
   return waitForTaskFinalStats(task, TaskState::kFailed, maxWaitMicros);
}

bool waitForTaskAborted(exec::Task* task, uint64_t maxWaitMicros) {
   return waitForTaskFinalStats(task, TaskState::kAborted, maxWaitMicros);
}
```